core-bundle/src/Resources/config/services.yml

```diff
@@ -1020,6 +1021,7 @@ services:
     Contao\CoreBundle\Intl\Locales:
         alias: contao.intl.locales
         public: true # backwards compatibility
+    Contao\CoreBundle\Mailer\AvailableTransports: '@contao.mailer.available_transports'
```

Comment:
shouldn't the alias also be public (if it was previously)?

Reply:
See #4027